src/redux/Characters/CharactersRedux.js

```diff
@@ -0,0 +1,40 @@
+import connectCharApi from '../Api/characterApi';
+
+const CHARACTERS = 'redux/actions/get_characters';
+const NAME = 'redux/actions/get_characters/name';
+
+const initialState = {
+  name: '',
+  chars: [],
+};
+
+const characters = (character, type) => ({
+  type,
+  payload: character,
+});
+
+export const getCharacter = ({ page, dispatch, name = '' }) => {
+  connectCharApi(page, name).then(({ results }) => {
+    dispatch(characters(name, NAME));
+    dispatch(characters(results, CHARACTERS));
+  }).catch(() => {});
+};
+
+/* export const getName = (name, dispatch) => {
+  connectCharApi(name).then(({ search }) => {
+    dispatch(characters(search));
+  });
+}; */
```

Comment:
- [ ] Please remove the commented code to make your code cleaner and more readable